tests/test_pipelines_text2text_generation.py

```diff
@@ -43,12 +43,15 @@ def run_pipeline_test(self, generator, _):
         with self.assertRaises(ValueError):
             generator(4)
 
-    @require_torch
-    def test_small_model_pt(self):
-        generator = pipeline("text2text-generation", model="patrickvonplaten/t5-tiny-random", framework="pt")
-        # do_sample=False necessary for reproducibility
-        outputs = generator("Something there", do_sample=False)
-        self.assertEqual(outputs, [{"generated_text": ""}])
+	@require_torch
+	def test_small_model_pt(self):
+		num_return_sequences = 3
+		generator = pipeline("text2text-generation", model="patrickvonplaten/t5-tiny-random", framework="pt")
+		# do_sample=False necessary for reproducibility
+		outputs = generator("Something there", do_sample=False,
+		                    num_return_sequences=num_return_sequences,
+		                    num_beams=num_return_sequences)
+		self.assertEqual(outputs, [{"generated_text": ANY(str)} for _ in range(num_return_sequences)])
```

Comment:
This needs to be added, not replacing the old test.

The default behavior is still important.
Also this test tests a real model (even though it's a random tiny) which means we cannot use ANY, we really need to test the actual value outputted by the model.

If the outputs are random with `do_sample` being used then there's likely a bug.

Reply:
Thank you for your response. I added a test after the old test. But check_code_quality fails.
Can the mods guide me on how to improve that? CircleCI says :


> black--check examples tests src utils
would reformat tests/test_pipelines_text2text_generation.py
Oh no! 💥 💔 💥
1 file would be reformatted, 1149 files would be left unchanged.
Exited with code exit status 1
CircleCI received exit code 1`

Reply:
@enze5088 

Usually `make fixup` should fix all the quality problems for you.